cl/search/management/commands/clean_docket_number_raw.py

```diff
@@ -0,0 +1,53 @@
+from cl.lib.command_utils import VerboseCommand, logger
+from cl.search.models import Docket
+from cl.search.signals import clean_docket_number_raw_and_update_redis_cache
+
+
+class Command(VerboseCommand):
+    help = "Clean docket_number_raw and send to LLM cleaning daemon if needed."
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.throttle = None
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--court-ids",
+            nargs="+",
+            type=str,
+            help="The court IDs to filter Dockets to update, may be more than one.",
+        )
+
```

Comment:
Do you have an idea of how many dockets we’ll process with this task? If we expect the command to run for several days, I’d suggest adding an `--auto-resume` argument so that if it crashes, it’s easier to resume from where it left off.

You can copy the `--auto-resume` mechanism from the `generate_opinion_embeddings` command. You’ll just need to use a different `compose_redis_key` to store the progress for this command.